index.bs

```diff
@@ -11,7 +11,7 @@ Notes:
 Title: Web Authentication: An API for accessing Public Key Credentials - Level
 Status: ED
 Prepare for TR: true
-TR: https://www.w3.org/TR/webauthn/
+TR: https://www.w3.org/TR/webauthn-2/
```

Comment:
Probably want to hold off merging until     https://www.w3.org/TR/webauthn-2/  is a valid link.

Reply:
SGTM

Reply:
agreed

Reply:
@wseltzer  ?

Reply:
@jcjones This was reverted by #1186 (now merged into this); #1187 will revert the revert and restore the "-2".

Reply:
AFAICT this PR now lacks the `-2` addition @jcjones is objecting to, and he sez essentially that below https://github.com/w3c/webauthn/pull/1161#issuecomment-474941638

so ISTM we can merge this once he clears this changes-requested review, OR since we've done what he asks and he's away for a few weeks, we can "dismiss (his) review" and merge it (I vote for doing the latter).